apps/opik-backend/entrypoint.sh

```diff
@@ -5,12 +5,12 @@ echo $(pwd)
 jwebserver -d /opt/opik/redoc -b 0.0.0.0 -p 3003 &
 
 echo "OPIK_VERSION=$OPIK_VERSION"
-echo "NEW_RELIC_ENABLED=$NEW_RELIC_ENABLED"
-echo "NEW_RELIC_VERSION=$NEW_RELIC_VERSION"
+echo "OTEL_SDK_DISABLED=$OTEL_SDK_DISABLED"
+echo "OTEL_VERSION=$OTEL_VERSION"
 
-if [[ "${NEW_RELIC_ENABLED}" == "true" && "${NEW_RELIC_LICENSE_KEY}" != "" ]];then 
-    curl -o /tmp/newrelic-agent.jar https://download.newrelic.com/newrelic/java-agent/newrelic-agent/${NEW_RELIC_VERSION}/newrelic-agent-${NEW_RELIC_VERSION}.jar
-    JAVA_OPTS="$JAVA_OPTS -javaagent:/tmp/newrelic-agent.jar"
+if [[ "${OTEL_SDK_DISABLED}" == "false" ]];then
```

Comment:
@thiagohora I rather not have `DISABLED` in the name 
can we rename to `OTEL_ENABLED` (since we download the sdk and enable it in our code)
default to false

Reply:
Ok, I was reusing the official variable from open telemetry: https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#general-sdk-configuration

But I can use our variable to invert the open telemetry variable.